test/jdk/sun/security/rsa/SpecTest.java

```diff
@@ -30,7 +30,7 @@
  * @run main SpecTest 512
  * @run main SpecTest 768
  * @run main SpecTest 1024
- * @run main SpecTest 1024 3
+ * @run main SpecTest 1024 65537
```

Comment:
I want to stop using F0 since it's no longer deemed valid for FIPS 186-4. For backward compatibility, we don't reject F0, but perhaps we should stop using it so people will start shifting to use F4. I can replace 3 with 167971.